### ISSUE TITLE
switchdiscover did not work on ubuntu18.4 and rhels6.10 from 20180813

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5518]
``switchdiscover`` did not work on ubuntu18.4 and rhels6.10 from 20180813.

The output change to 

```
RUN:switchdiscover --range 10.4-5.24-25.1 [Wed Aug 15 09:38:22 2018]
ElapsedTime:0 sec
RETURN rc = 0
OUTPUT:
Error: Unsupported request
```
But this command can work correctly on other operating systems.

**Comment by cxhong** [comment]
@hu-weihua  @zet809 ,  I can take look this.  do u have system has Ubuntu 18.4 and rhels6.10?

**Comment by zet809** [comment]
Hi, @cxhong , this issue have been fixed with my PR #5519

**Comment by hu-weihua** [comment]
Have been verified in automation test environment. thanks @zet809 and @cxhong .
